templates/tools/install.hbs

```diff
@@ -40,7 +40,9 @@
         cross-compilation targets.
       </p>
       <p>
-        For more information see the
+        If you've installed <code>rustup</code> in the past, you can update
```

Comment:
One suggestion: let's put this in its own paragraph, as the other blocks are.

```hbs
<p>
  If you've installed <code>rustup</code> in the past, you can update
  your installation by running <code>rustup upgrade</code>.
</p>
```

Then we can leave the original in place.

Reply:
Sure, if you prefer. Updated.

Reply:
Thank you!